### DISCUSSION TITLE
why playing osu!lazer using wsa (windows subsystem for android) is laggy?

### DISCUSSION THREAD

**Discussion by catlomao** [ppy/osu#28043] [Q&A] [answered]
Getting low fps + higher latency using wsa (windows subsystem for android) android version of osu!lazer, slower beatmap loading,low audio quality, wsa android version is 13 (tiramisu) the export log button somehow is not visable i cant export log lol

https://github.com/ppy/osu/assets/84031819/22a6e9ee-7c91-47eb-abab-9adec6a7de79

**Comment by Joehuu** [comment, marked as answer]
Is there a reason you didn't install lazer natively on Windows?

I would call this setup unsupported anyway due to the above and [Microsoft is ending support for the Windows Subsystem for Android](https://learn.microsoft.com/en-us/windows/android/wsa/).

**Reply by catlomao** [reply]
just to see if it works, wsa is not that great anyways
